README.md

```diff
@@ -20,6 +20,13 @@ http://marketplace.eclipse.org/content/jbc/
 
 ## Usage
 
+When first time opening Editor within a project, a dialogue will appear with suggestion to convert to XText project.  
```

Comment:
The correct notation is "Xtext"
See https://blogs.itemis.com/en/about-the-notation-of-the-names-xtext-xtend-xbase-xcore-xpand